src/main/datatypes/MQ2AdvLootType.cpp

```diff
@@ -345,7 +347,43 @@ bool MQ2AdvLootItemType::GetMember(MQVarPtr VarPtr, const char* Member, char* In
 		Dest.Type = pBoolType;
 		return true;
 
-	default: break;
+	case AdvLootItemMembers::Status:
+	{
+		Dest.Type = pStringType;
+
+		if( itemList == pAdvancedLootWnd->pPLootList )
+		{
+			Dest.Ptr = "INVALID";
+			return true;
+		}
+		switch( item.Unknown0x64 )
+		{
+		case 0:
+			Dest.Ptr = "WAITING";
+			return true;
+		case 1:
+			Dest.Ptr = "ASKING";
+			return true;
+		case 2:
+			Dest.Ptr = "ROLLING";
+			return true;
+		case 3:
+			Dest.Ptr = "STOPPED";
+			return true;
+		case 4:
+			Dest.Ptr = "CLICKROLL";
+			return true;
+		case 5:
+			Dest.Ptr = "FREEGRAB";
+			return true;
+		default:
+			Dest.Ptr = "UNKNOWN";
+			return true;
+		}
```

Comment:
You can't set pointers to string literals, you need to copy into a pointer and then set it (we use DataTypeTemp). Look at CharacterType.CombatState for an example.